### ISSUE TITLE
SSLV3 error

### ISSUE THREAD

**Issue by jayanth1991** [websocket-client/websocket-client#336]
My client side got this error and disconnected from the websocket. My Tornado server is behind a Nginx server.
```py
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/ubiqutils/controllers/maincontroller.py", line 516, in state_update
    ws.send(json.dumps(resp))
  File "/usr/local/lib/python2.7/dist-packages/websocket/_app.py", line 115, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 239, in send
    return self.send_frame(frame)
  File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 264, in send_frame
    l = self._send(data)
  File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 425, in _send
    return send(self.sock, data)
  File "/usr/local/lib/python2.7/dist-packages/websocket/_socket.py", line 112, in send
    return sock.send(data)
  File "/usr/lib/python2.7/ssl.py", line 705, in send
    v = self._sslobj.write(data)
error: [Errno 14] Bad address
ERROR test_websocket.py on_error:157: [SSL: SSLV3_ALERT_BAD_RECORD_MAC] sslv3 alert bad record mac (_ssl.c:1754)
```

This has happened across multiple client machines, not sure what the root cause is...

**Comment by jayanth1991** [comment]
Any help with this?

Do I need to change any configuration on Nginx?

**Comment by engn33r** [comment]
I think this is the same issue that got fixed by #353. If that is not the case, you can add additional detail here or open a new issue.
